package.json

```diff
@@ -28,12 +27,10 @@
   },
   "homepage": "https://github.com/sealink/money_beans#readme",
   "devDependencies": {
-    "coffee-script": "^1.10.0",
-    "karma": "^1.1.0",
-    "karma-coffee-preprocessor": "^1.0.0",
-    "karma-jasmine": "^1.0.2",
-    "karma-phantomjs-launcher": "^1.0.1",
-    "jasmine-core": "^2.4.1",
-    "phantomjs-prebuilt": "^2.1.7"
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "expect.js": "^0.3.1",
```

Comment:
Out of interest why expect.js over something like chai/expect which seems to be more common practice?

Reply:
Happy to change. I believe I choose it because it more closely matched the syntax.